GitUI/Plugin/GitPluginSettingsContainer.cs

```diff
@@ -35,5 +36,15 @@ public override void SetValue<T>(string name, T value, Func<T, string> encode)
         {
             ExternalSettings.SetValue(_pluginName + name, value, encode);
         }
+
+        public override void SetCredential(string name, NetworkCredential value, IGitModule gitModule)
+        {
+            ExternalSettings.SetCredential(name, value, gitModule);
+        }
+
+        public override NetworkCredential GetCredential(string name, IGitModule gitModule, NetworkCredential defaultValue)
```

Comment:
```suggestion
        public override NetworkCredential GetCredentials(string name, IGitModule gitModule, NetworkCredential defaultValue)
```

Reply:
It is not correct, because method return one credential, but not set of credentials